deployment/k8s/charts/interoperator/conf/settings.yaml

```diff
@@ -0,0 +1,118 @@
+# This file should not be used in deployed environments. Instead, set
```

Comment:
For remotely hosted helm repo, this settings file kinda becomes rigid and no more a configuration. Is there any way some values for this settings can flow from `values.yaml` so that user has flexibility to override defaults in `helm install` command